### ISSUE TITLE
Potential Double Dropout of MLP

### ISSUE THREAD

**Issue by witherhoard99** [nicklashansen/dreamer4#8]
In the MLP class, dropout is applied
```py
def forward(self, x: torch.Tensor) -> torch.Tensor:
        u, v = self.fc_in(x).chunk(2, dim=-1)
        h = u * F.silu(v)
        h = self.drop(h)
        y = self.fc_out(h)
        y = self.drop(y)
        return y
```

In the block causal layer, there is dropout applied after the MLP again
```py
def forward(self, x: torch.Tensor) -> torch.Tensor:
        x = x + self.drop1(self.space(self.norm1(x)))
        if self.do_time:
            x = x + self.drop2(self.time(self.norm2(x)))
        x = x + self.drop3(self.mlp(self.norm3(x))) # This line here
        return x
```

If I understand the code correctly, this means that dropout is applied twice. With the potential for high masking (up to 0.9) and `nn.Dropout`'s inverted dropout scaling all gradients by 1.1x every single BlockCausalLayer, this likely harms convergence (default dropout of 0.05 starts behaving close to 0.1).

**Comment by nicklashansen** [comment]
Thanks for raising this issue, that's a great catch! I'm a bit caught up with other things at the moment but I'll try to verify this soon enough.

**Comment by nicklashansen** [comment]
Commit 5dd1f0f980d61cba567c3fe41632e3800d0cd5ac removes the additional dropout, thanks for bringing it to my attention! In my very preliminary tests I do not see any notable change in training performance but it's good to address it regardless.
